content/news/christmas-2017.md

```diff
@@ -0,0 +1,16 @@
+
+This year, we have to start with some really unfortunate news. Santa's sleight carrying the next OpenRA release was shot down by an AA gun!
+But don't worry, we do our best and work hard towards recovering something playable for the impending next playtest.
+
+In the meantime, we are happy to see that you still enjoy our latest work. With release-20171014 we reached an impressive total of half a million online games played since late 2014.
```

Comment:
This may end up delaying the post too much, so we could just have a short note and the screenshot?